packages/world/src/systems/BedSystem.sol

```diff
@@ -87,10 +90,12 @@ contract BedSystem is System {
     bedEntityId = bedEntityId.baseEntityId();
     require(!BedPlayer._getPlayerEntityId(bedEntityId).exists(), "Bed full");
 
+    // TODO: do we need a forcefield for sleeping?
```

Comment:
I think it makes sense to only allow sleeping in charged force fields. If we dont require a charged force field for sleeping, where would the player energy drain from?

Reply:
sg, let's add on a different pr, will create an issue

Reply:
another option would be to just warn the user instead I guess, that way we keep things simple